src/settings/utils/time.js

```diff
@@ -1,9 +1,29 @@
 import moment from 'moment';
 
+import {
+  localizer,
+  MONDAY,
+  TUESDAY,
+  WEDNESDAY,
+  THURSDAY,
+  FRIDAY,
+  SATURDAY,
+  SUNDAY,
+} from '../constants';
+
 // eslint-disable-next-line import/prefer-default-export
 export const setTimezoneOffset = (date) => {
   const momentDate = moment(date);
   const utcOffset = -(momentDate.utcOffset() / 60);
 
   return momentDate.add(utcOffset, 'hours');
 };
+
+export const getWeekDays = (locale) => {
+  const weekDays = [MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY];
+  const method = (!locale || localizer.startOfWeek(locale) === 0) ? 'unshift' : 'push';
+
+  weekDays[method](SUNDAY);
+
+  return weekDays;
+};
```

Comment:
Please add a function header explaining what this function does, including what the format of the `locale` argument is (e.g. `en`, `en_US`, `en-US`, etc.) and what shape the return value has.

Reply:
done. thanks

Reply:
Also I've added support for any first day of week(it can be `Sunday`, `Monday` or `Saturday` or any other day)